test/TestAnsibleLintRule.py

```diff
@@ -0,0 +1,7 @@
+from ansiblelint.rules import AnsibleLintRule
+
+
+def test_unjinja():
```

Comment:
I'd prefer this to have a docstring but low priority

Reply:
Also, ideally, there could be multiple cases to document more behaviors as separate tests.